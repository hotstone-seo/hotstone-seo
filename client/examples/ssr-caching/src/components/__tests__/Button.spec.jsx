import React from 'react';
import Button from '../Button.jsx';

test('Should render Button correctly', () => {
  const tree = render(<Button />);

  expect(tree).toMatchSnapshot();
});
