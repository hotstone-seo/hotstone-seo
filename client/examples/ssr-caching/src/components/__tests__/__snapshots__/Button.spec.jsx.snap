// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Button correctly 1`] = `
<a
  class="btn"
/>
`;
